src/store/actions/employees.js

```diff
@@ -0,0 +1,46 @@
+import * as actionTypes from './actionTypes';
+
+import { getAllUsers } from '../../services/userService';
+
+const _fetchAllEmployees = () => {
+  return {
+    type: actionTypes.FETCHING_EMPLOYEES,
+  };
+};
+
+const _fetchAllEmployeesFailed = error => {
+  return {
+    type: actionTypes.FETCHING_EMPLOYEES_FAILURE,
+    error,
+    resultCount: 0,
+  };
+};
+
+const _receiveAllEmployees = (employees, count) => {
```

Comment:
`fetchAllEmployessSuccess` and `resultCount` :)

Reply:
Done 👍